### PR TITLE
Restructure NATS client

### PR DESCRIPTION
**Pull request by ccorley**
Minor restructure of NATS client code, especially around creating subscribers, to reduce code.

Signed-off-by: ccorley <ccorley@us.ibm.com>

### CODE REVIEW

File: connect/clients/nats.py [hunk 1]
```diff
@@ -42,8 +42,8 @@ async def create_nats_subscribers():
     Create NATS subscribers.  Add additional subscribers as needed.
     """
     await start_sync_event_subscribers()
-    await start_timing_subscriber()
-    await start_retransmit_subscriber()
+    await start_subscriber("TIMING", nats_timing_event_handler)
```
Comment (dixonwhitmire) [inline connect/clients/nats.py:45]: Should we add a config for TIMING ?

**Review by dixonwhitmire — APPROVED** [review]
Looks good @ccorley ! Less code that does the same stuff is always a win! I just had a minor question regarding a config for the TIMING subject.

File: connect/clients/nats.py (hunk 1 above)
Comment (ccorley) [reply connect/clients/nats.py:45]: We decided (at the time) to make internal subjects literals and not configurable.  We can revisit if needed.

Comment (dixonwhitmire) [reply connect/clients/nats.py:45]: Gotcha. That make sense. We don't really want to override these with an environment variable. Thanks for the reminder!

**Review by dixonwhitmire — APPROVED** [review]

**Pull request merged**